shell/pages/index.vue

```diff
@@ -19,6 +21,11 @@ export default {
     }
 
     const afterLoginRouteObject = store.getters['prefs/afterLoginRoute'];
+    const targetRoute = app.router.resolve(afterLoginRouteObject);
+
+    if (targetRoute?.route?.fullPath === route.fullPath) {
+      return redirect(dashboardHome);
```

Comment:
This does not make sense - why would we redirect to the dashboard home if the login route is the current route? Should this not just ignore and return?

Reply:
@ly5156 ^^ can you explain the intent here?

Reply:
okay - understand this now and made a small change